data/tutorials/language/0it_01_basic_datatypes.md

```diff
@@ -705,6 +705,14 @@ val ghorghor_bey : character =
 - : int = 17
 ```
 
+To construct a new record with some field values changed without typing in the unchanged fields we can use record update syntax as shown:
```

Comment:
```suggestion
We can use record update syntax to construct a new record. This is useful when you need to change some field values without typing in the unchanged fields, as shown below:
```

Do you think this flows better (and is it still accurate)?
Also, is record update a type of syntax, so perhaps _record update_ syntax?

Reply:
I think saying that we construct a new record first is better because the effect is that there's a new record. When we mention "record update syntax" first, people may think that we are changing the existing value.